fis/ut/fis_ut.c

```diff
@@ -74,9 +73,9 @@ static void fis_ut_motr_stop(struct m0_rpc_server_ctx *rctx)
 
 static void fis_ut_client_start(void)
 {
-	int rc;
+	int                      rc;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated